content/1e/rules/character-creation.de.mdx

```diff
@@ -0,0 +1,275 @@
+- Zauberwirken von Magiern und Nekromanten
+- Fertigkeitswürfe um den Wert eines Gegenstands einzuschätzen, ein Schriftstück zu entschlüsseln, mechanische Fallen unschädlich zu machen,
+Dokumente anzufertigen, gelerntes Wissen abzurufen, Magie zu identifizieren oder ähnliches
+
+#### Weisheit (Wei)
+
+Weisheit bestimmt die Intuition, die Einsicht und das Wahrnehmungsvermögen.
+Leider scheint dies dem Unheiligen genauso zu dienen wie dem Heiligen — vielleicht sogar mehr.
+Während Intelligenz die Fähigkeit repräsentiert Informationen zu analysieren,
+bedeutet Weisheit, mit der Umgebung im Einklang zu sein und sie bewusst wahrzunehmen.
+Wenn ein Charakter scharfe Sinne haben soll, benötigt er einen hohen Weisheitswert.
+
+Der Weisheit-Modifikator eines Charakters wird beeinflusst Folgendes:
+
+- Geistiger Widerstand (GW), zusammen mit Intelligenz und Charisma
+- Rüstungsklasse (RK), zusammen mit Konstitution und Geschick
+- [Stabilisieren][] eines bewusstlosen Verbündeten
+- Zauberwirken von Klerikern und Druiden
+- Fertigkeitswürfe um Dinge mit den Sinnen wahrzunehmen, die Motive eines NSCs zu erkennen, Verwundete zu heilen, den Weg zu finden oder ähnliches
+
+#### Charisma (Cha)
+
+Charisma bestimmt die Ausstrahlung der Persönlichkeit und das soziale Auftreten. Schwer zu definieren, doch nicht zu übersehen, wenn es einem begegnet.
+Charisma kann Überzeugungskraft, persönliche Anziehungskraft, Führungsqualität und körperliche Attraktivität sein.
+Diese Fertigkeit repräsentiert die tatsächliche Stärke der Persönlichkeit, nicht nur, wie andere einen im sozialen Umfeld wahrnehmen.
+
+Der Charisma-Modifikator eines Charakters beeinflusst Folgendes:
+
+- Geistiger Widerstand (GW), zusammen mit Intelligenz und Weisheit
+- Zauberwirken von Zauberern
+- Paladin Klassenmerkmale
+- Fertigkeitswürfe um Personen zu täuschen oder zu beeinflussen, diplomatisch zu sein,
+so zu tun, als wäre man jemand anderes, Informationen durch Herumfragen zu sammeln, mit Tieren umzugehen,
+Personen einzuschüchtern, indem man ihre Schwachstellen ausnutzt, Leute mit darstellenden Künsten zu unterhalten oder ähnliches
+
+### Attribute auswürfeln
+
+Es werden 4d6 für jedes der sechs Attribute (Stärke, Konstitution, Geschick, Intelligenz, Weisheit und Charisma) gewürfelt.
+Der niedrigste Würfel bei jedem Wurf fällt weg. Die Ergebnisse werden frei verteilt, wie sie am besten zu dem Charakter passen.
+
+(Diese Methode führt zu einem Durchschnitt von 12,24, was bei 6 Fertigkeiten eine Gesamtzahl von 73,44 ergibt.)
+
+### Attribute kaufen
+
+Es gibt 28 Punkte mit denen die Attribute nach der unten stehenden Tabelle gekauft werden.
+
+| Fertigkeit | Kosten |
+| -------    | ----   |
+| 18         | 16     |
+| 17         | 13     |
+| 16         | 10     |
+| 15         | 8      |
+| 14         | 6      |
+| 13         | 5      |
+| 12         | 4      |
+| 11         | 3      |
+| 10         | 2      |
+| 9          | 1      |
+| 8          | 0      |
```

Comment:
Can we include the modifier in the table here? I added it the other day to the English version because it was mentioned by someone that it was missing.

Here is the English version of the table for reference: https://github.com/13vaults/13vaults.com/blob/main/content/1e/rules/character-creation.mdx?plain=1#L106-L118